### PR TITLE
PPTT-2755 Renamed backend tracker & friends to CPP

### PR DESCRIPTION
**Pull request by sebastianpiskorski**

### CODE REVIEW

File: data_collection/web/guides.rst [hunk 1]
```diff
@@ -367,7 +367,7 @@ As you can imagine this may produce inconsistent results, providing those values
 Manual content tracking
 ^^^^^^^^^^^^^^^^^^^^^^^
 
-If for some reason automatic content tracking does not suit you needs you may still trigger :ref:`trackContentImpression<jtc-api-trackContentImpression>` and :ref:`trackContentInteraction<jtc-api-trackContentInteraction>` JS Tracking Client functions manually.
+If for some reason automatic content tracking does not suit you needs you may still trigger :ref:`trackContentImpression<jtc-api-trackContentImpression>` and :ref:`trackContentInteraction<jtc-api-trackContentInteraction>` JavaScript Client functions manually.
```
Comment (adaqus) [inline data_collection/web/guides.rst:370]:
```suggestion
If for some reason automatic content tracking does not suit you needs you may still trigger :ref:`trackContentImpression<jtc-api-trackContentImpression>` and :ref:`trackContentInteraction<jtc-api-trackContentInteraction>` JavaScript Tracking Client functions manually.
```

File: glossary.rst [hunk 2]
```diff
@@ -62,6 +62,9 @@ Glossary
         .. note::
             If :term:`analytics attribute` uses the same name - it will be represented as a separate :term:`attribute`.
 
+    Collecting & Processing Pipeline (CPP)
+        Formerly a trakcer backend. A set of services that is able to receive, store and process requests from JavaScript Tracking Client. Requests processed by CPP are passed further for Reporting services.
```
Comment (adaqus) [inline glossary.rst:66]:
```suggestion
        Formerly called tracker backend. A set of services that are able to receive, store and process requests from various tracking clients (e.g. JavaScript Tracking Client, SDKs). Requests processed by CPP are passed further to Reporting services.
```

**Review by adaqus — APPROVED** [review]

**Comment by sebastianpiskorski** [comment]
@adaqus I was wondering about these three additional renames:
```
Tracker debugger API -> CPP debugger API ?
Tracking HTTP API -> CPP HTTP API ?
Tracker Settings API -> CPP Settings API
```

What do you think?

**Comment by adaqus** [comment]
> @adaqus I was wondering about these three additional renames:
> 
> ```
> Tracker debugger API -> CPP debugger API ?
> Tracking HTTP API -> CPP HTTP API ?
> Tracker Settings API -> CPP Settings API
> ```
> 
> What do you think?

`CPP HTTP API` does not inform what the API does. `CPP Tracking API` would be nicer.

File: data_collection/web/guides.rst (hunk 1 above)
Comment (sebastianpiskorski) [reply data_collection/web/guides.rst:370]: changed

**Review by adaqus — APPROVED** [review]

**Review by RomanSek — APPROVED** [review]

File: data_collection/api/tracker_debugger_api.rst [hunk 3]
```diff
@@ -1,6 +1,6 @@
-Tracker debugger API
-====================
-Tracker debugger API exposes sessions in live mode. It's a useful tool for verifying tracking code implementation and observing changes done to it.
+CPP debugger API
+================
+CPP debugger API exposes sessions in live mode. It's a useful tool for verifying JavaScript Tracking Snippet implementation and observing changes done to it.
```
[the remark this answers is not in the document]
Comment (sebastianpiskorski) [reply data_collection/api/tracker_debugger_api.rst:3]: `JavaScript Tracking Client Snippet` is a mistake.

[the remark this answers is not in the document]
Comment (sebastianpiskorski) [reply data_collection/api/tracker_debugger_api.rst:3]: I've only found this: https://github.com/PiwikPRO/PPMS-PublicDocs/pull/663/files#diff-2dad6b29e112e2679b09221ea5763812fba9055810923f371552cab6e6073373L21

It was already renamed.

**Pull request merged**